agent/agentcontainers/devcontainercli_test.go

```diff
@@ -229,6 +230,12 @@ func TestDockerDevcontainerCLI(t *testing.T) {
 	if os.Getenv("CODER_TEST_USE_DOCKER") != "1" {
 		t.Skip("skipping Docker test; set CODER_TEST_USE_DOCKER=1 to run")
 	}
+	if runtime.GOOS != "linux" {
+		t.Skip("Skipping on non-Linux OS")
+	}
+	if _, err := exec.LookPath("devcontainer"); err != nil {
+		t.Fatal("this test requires the devcontainer CLI: npm install -g @devcontainers/cli")
+	}
```

Comment:
review: I didn't look too closely into what was actually going on here, but I suspect it's related to path differences between the host and the Docker VM on MacOS/Windows.

Reply:
I've run this on macOS as well, so IMO we could allow that, but agree that we need to verify the devcontainer CLI here, good add 👍🏻